tests/newtype.rs

```diff
@@ -0,0 +1,68 @@
+#[cfg(all(test, feature = "iterator", feature = "macro"))]
+mod test {
+    use cosmwasm_std::{testing::MockStorage, Addr, Uint128};
+    use cw_storage_macro::NewTypeKey;
+    use cw_storage_plus::Map;
+    use derive_more::Display;
+    use serde::{Deserialize, Serialize};
+
+    #[test]
+    fn newtype_compiles() {
+        #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, NewTypeKey)]
+        struct TestKey(u64);
+
+        let _ = TestKey(100);
+    }
+
+    #[test]
+    fn newtype_works() {
```

Comment:
My instinct with tests would be to verify a newtype behaves the same as the inner key, but it's not a hill I'd die on

Reply:
nice idea! done in https://github.com/CosmWasm/cw-storage-plus/pull/88/commits/3b0833c54232f69412c0ac8cefae30e9e5c61a1a